### DISCUSSION TITLE
Is milvus 2 supporting GPU based index (i have a faiss index of 35 gb size) ?

### DISCUSSION THREAD

**Discussion by astar10239** [milvus-io/milvus#16379] [Q&A and General discussion]
Is milvus 2.0 latest version supporting GPU based index ?

I have a huge faiss index of 35 gb size and wanted to load in 2 gpus for faster mili-sec search

**Comment by xiaofan-luan** [comment]
> Is milvus 2.0 latest version supporting GPU based index ?
> 
> I have a huge faiss index of 35 gb size and wanted to load in 2 gpus for faster mili-sec search

Not yet. But we have a plan to support GPU on 2.2(Targeting July but might be ready before taht)
So far you can use Milvus 1.x or use Milvus 2.0 without GPU.
Mili-sec search should be possible for only 35Gb data

**Reply by studygogogo1** [reply]
Excuse me， When will the milvus that supported GPU be released？

**Reply by KirylKovaliov** [reply]
any updates on this ?

**Reply by yhmo** [reply]
> any updates on this ?

The work of v2.2 is not yet begin. Now we are focusing on stability issues of v2.1, in the next two months we will release several minor versions of v2.1. After the v2.1 is stable enough, we will begin to build v2.2.

**Comment by jayshree346** [comment]
Hello,
I have two queries to ask
1) Now  we tried to use milvus gpu version 1.0.0 , enable to install compatible pymilvus. Can you please suggest which specific version of pymilvus and milvus_cli we can install for milvus gpu 1.0.0.  

2) When can we expect the milvus 2 supporting GPU based index searching?

**Reply by yhmo** [reply]
> Hello, I have two queries to ask
> 
> 1. Now  we tried to use milvus gpu version 1.0.0 , enable to install compatible pymilvus. Can you please suggest which specific version of pymilvus and milvus_cli we can install for milvus gpu 1.0.0.
> 2. When can we expect the milvus 2 supporting GPU based index searching?

The pymilvus v1.0.1 is for milvus v1.0.0.
The milvus v1.1.1 is stable than v1.0.0,  pymilvus=1.1.2
https://github.com/milvus-io/pymilvus/tree/1.x

In next 2~3 month, we will focus on stability of  the milvus v2.x. New features will be postponed, including the GPU supporting.
